neo/SmartContract/ApplicationEngine.cs

```diff
@@ -1,22 +1,18 @@
 using Neo.Ledger;
 using Neo.Network.P2P.Payloads;
 using Neo.Persistence;
-using Neo.SmartContract.Manifest;
-using Neo.SmartContract.Native;
 using Neo.VM;
 using System;
 using System.Collections.Generic;
-using VMArray = Neo.VM.Types.Array;
 
 namespace Neo.SmartContract
 {
-    public class ApplicationEngine : ExecutionEngine
+    public partial class ApplicationEngine : ExecutionEngine
     {
         public static event EventHandler<NotifyEventArgs> Notify;
         public static event EventHandler<LogEventArgs> Log;
 
-        public static readonly long GasFree = 10 * (long)NativeContract.GAS.Factor;
-        private const long ratio = 100000;
+        public const long GasFree = 0;
```

Comment:
This means that we will lose the free transactions?

Reply:
Yes. The base price has been greatly reduced, and free transactions are no longer allowed. This will increase the demand for gas.

Reply:
Maybe we can reduce it from 10 to 1 or 2, but remove it...  we should know the impact of this

Reply:
This can be discussed. If we decide to keep free, we can just change the number above. That's why I keep the field and set it to 0 instead of just remove it. This PR is a structure, all actual prices can be discussed later.